src/components/PostEditor/PostEditor.connector.js

```diff
@@ -125,9 +130,9 @@ export const mergeProps = (stateProps, dispatchProps, ownProps) => {
   const goToPost = createPostAction => {
     const id = get('payload.data.createPost.id', createPostAction)
     // * The single letter params are used in the Stream and elsewhere
-    // and translate as follow: `s`(ort), `t`(ab), `q`(uery/search)
+    // and translate as follow: `s`(ort), `t`(ab), `q`(uery/'search')
     // The remaining whitelisted params are for the map view.
-    const querystringWhitelist = ['s', 't', 'q', 'zoom', 'center', 'lat', 'lng']
+    const querystringWhitelist = ['s', 't', 'q', 'search', 'zoom', 'center', 'lat', 'lng']
```

Comment:
This isn't related to duplicate -- it seems we use 'search' instead of 'q'? Also, search is weird in that I couldn't figure out how to add "or" terms, it only searches on one term and it's the full term -- I assume I'm wrong, that's not the conventional way search works.